deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -58,3 +58,30 @@
     job: docker system prune -af
     state: "present"
   changed_when: False
+
+- name: Setup weekly NPM cleanup cron task
+  cron:
+    name: "npm-cleanup"
+    user: "{{ jenkins_name }}"
+    special_time: "weekly"
+    job: find /var/lib/jenkins/workspace/ -path "*/.node_modules/*" -delete
+    state: "present"
+  changed_when: False
```

Comment:
I don't have an explanation for the `changed_when: False` entries. I see that they were in the existing `cron` entries, but it would be good to identify why they're there and remove them if we can't determine why.

Might need to loop in @tnation14 because I think he added the original and may have an explanation.

Reply:
>changed_when marks the task as changed based on the evaluation of the conditional statement which in your case is:
>```
>"command_result.stdout == 'Ignoring failure...'"
>```
>So whenever this condition is true, the task will be marked as changed.

So, my understanding is that this will never update the cron tasks again?

Reply:
`changed_when`  doesn't actually impact if the task _runs_, it just keeps Ansible from showing that a task has _changed_ (see [docs](https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html#overriding-the-changed-result)). I added it here to make `molecule` tests happy. There's some more discusson on this in https://github.com/raster-foundry/raster-foundry/pull/522#issuecomment-253572407